js/ojsxc.js

```diff
@@ -392,4 +398,63 @@
             observeContactsMenu();
         }
     });
+
+    $(document).on( "click", '#jsxc_roster p', function() {
+        if (jsxc.storage.getItem('serverType') === 'internal') {
+           startInternalBackend();
+        }
+    });
+
+    function startInternalBackend() {
+       jsxc.bid = OC.currentUser.toLowerCase() + '@' + window.location.host;
+
+        jsxc.storage.setItem('jid', jsxc.bid + '/internal');
+        jsxc.storage.setItem('sid', 'internal');
+        jsxc.storage.setItem('rid', '123456');
```

Comment:
I guess this is not required (should be handled by `jsxc.start`), but I will check it.